app/rdf/query-dimension-values.ts

```diff
@@ -545,7 +545,7 @@ export const loadMinMaxDimensionValues = async ({
 }) => {
   const query = SELECT`(MIN(?value) as ?minValue) (MAX(?value) as ?maxValue)`
     .WHERE`${iriToNode(datasetIri)} ${ns.cube.observationSet}/${ns.cube.observation} ?observation .
-?observation <${dimensionIri}> ?value .
+?observation ${dimensionIri} ?value .
```

Comment:
How come it already contains the brackets ?

Reply:
It was a `Term`, which gets converted to a `<>` pattern automatically when in `SELECT` context. I refactored this to pass string, so it's more clear 👍